website/docs/reference/resource-configs/snowflake-configs.md

```diff
@@ -4,6 +4,21 @@ id: "snowflake-configs"
 description: "Snowflake Configurations - Read this in-depth guide to learn about configurations in dbt."
 ---
 
+| Configuration    | Description    | 
+|------------------|----------------|
+|[Iceberg table format](/reference/resource-configs/snowflake-configs#iceberg-table-format)|The dbt-snowflake adapter supports the Iceberg table format and is available for three of the Snowflake materializations [table](/docs/build/materializations#table), [incremental](/docs/build/materializations#incremental) and [dynamic tables](/reference/resource-configs/snowflake-configs#dynamic-tables)|
+|[Dynamic tables](/reference/resource-configs/snowflake-configs#dynamic-tables)|Specific to Snowflake but follows the implementation of [materialized views](/docs/build/materializations#Materialized-View).|
```

Comment:
can we give some more info here? 

e.g. use the Snowflake-specific dynamic materialization to create dynamic tables. Supports settings like target_lag, refresh_mode, and on_configuration_change.